spring-cloud-stream-app-descriptor/src/main/resources/1.0.0.BUILD-SNAPSHOT/kafka-apps-docker.properties

```diff
@@ -0,0 +1,46 @@
+source.syslog=docker://org.springframework.cloud.stream.app:syslog-source-kafka:1.0.0.M1
+source.tcp=docker://org.springframework.cloud.stream.app:tcp-source-kafka:@project.version@
+source.time=docker://org.springframework.cloud.stream.app:time-source-kafka:@project.version@
+source.trigger=docker://org.springframework.cloud.stream.app:trigger-source-kafka:1.0.0.M1
+source.triggertask=docker://org.springframework.cloud.stream.app:triggertask-source-kafka:@project.version@
+source.twitterstream=docker://org.springframework.cloud.stream.app:twitterstream-source-kafka:@project.version@
+processor.bridge=docker://org.springframework.cloud.stream.app:bridge-processor-kafka:@project.version@
+processor.filter=docker://org.springframework.cloud.stream.app:filter-processor-kafka:@project.version@
+processor.groovy-filter=docker://org.springframework.cloud.stream.app:groovy-filter-processor-kafka:@project.version@
+processor.groovy-transform=docker://org.springframework.cloud.stream.app:groovy-transform-processor-kafka:@project.version@
+processor.httpclient=docker://org.springframework.cloud.stream.app:httpclient-processor-kafka:@project.version@
+processor.pmml=docker://org.springframework.cloud.stream.app:pmml-processor-kafka:@project.version@
+processor.scriptable-transform=docker://org.springframework.cloud.stream.app:scriptable-transform-processor-kafka:@project.version@
+processor.splitter=docker://org.springframework.cloud.stream.app:splitter-processor-kafka:@project.version@
+processor.transform=docker://org.springframework.cloud.stream.app:transform-processor-kafka:@project.version@
+sink.aggregate-counter=docker://org.springframework.cloud.stream.app:aggregate-counter-sink-kafka:@project.version@
+sink.cassandra=docker://org.springframework.cloud.stream.app:cassandra-sink-kafka:1.0.0.M1
+sink.counter=docker://org.springframework.cloud.stream.app:counter-sink-kafka:@project.version@
+sink.field-value-counter=docker://org.springframework.cloud.stream.app:field-value-counter-sink-kafka:@project.version@
+sink.file=docker://org.springframework.cloud.stream.app:file-sink-kafka:@project.version@
+sink.ftp=docker://org.springframework.cloud.stream.app:ftp-sink-kafka:@project.version@
+sink.gemfire=docker://org.springframework.cloud.stream.app:gemfire-sink-kafka:1.0.0.M1
+sink.gpfdist=docker://org.springframework.cloud.stream.app:gpfdist-sink-kafka:1.0.0.M1
+sink.hdfs=docker://org.springframework.cloud.stream.app:hdfs-sink-kafka:@project.version@
+sink.hdfs-dataset=docker://org.springframework.cloud.stream.app:hdfs-dataset-sink-kafka:1.0.0.M1
+sink.jdbc=docker://org.springframework.cloud.stream.app:jdbc-sink-kafka:@project.version@
+sink.log=docker://org.springframework.cloud.stream.app:log-sink-kafka:@project.version@
+sink.rabbit=docker://org.springframework.cloud.stream.app:rabbit-sink-kafka:@project.version@
+sink.redis=docker://org.springframework.cloud.stream.app:redis-sink-kafka:1.0.0.M1
```

Comment:
let's rename this to `redis-messaging` or something?  It doesn't write to set like data structures. See https://github.com/spring-cloud/spring-cloud-stream-app-starters/blob/master/redis/spring-cloud-starter-stream-sink-redis/src/main/java/org/springframework/cloud/stream/app/redis/sink/RedisSinkConfiguration.java